src/ast/mod.rs

```diff
@@ -1,33 +1,39 @@
-use crate::pattern::Pattern;
+mod pattern;
+mod span;
+
+pub use pattern::*;
+pub use span::*;
+
 use crate::types::Value;
 use serde::{Deserialize, Serialize};
 
 #[derive(Debug, Deserialize, Serialize)]
 pub enum Expr<'a> {
-    Ident(&'a str),
-    Value(Value<'a>),
-    Equals(Box<Expr<'a>>, Box<Expr<'a>>),
-    NotEquals(Box<Expr<'a>>, Box<Expr<'a>>),
-    LessEquals(Box<Expr<'a>>, Box<Expr<'a>>),
-    LessThan(Box<Expr<'a>>, Box<Expr<'a>>),
-    GreaterThan(Box<Expr<'a>>, Box<Expr<'a>>),
-    GreaterEquals(Box<Expr<'a>>, Box<Expr<'a>>),
-    And(Box<Expr<'a>>, Box<Expr<'a>>),
-    Or(Box<Expr<'a>>, Box<Expr<'a>>),
+    #[serde(borrow)]
```

Comment:
Any reason for renaming these?

Reply:
It looks better when it all lines up :)

Reply:
Could it be confusing to have some of the shorter names like LT? I am not sure if some of the shorter names are too non-descriptive. What do you think?

Reply:
I think it's hard to misinterpret given the context.

Reply:
Would three letters be acceptable :wink: (e.g. LTh, GEq, NEq, Eql)?

Reply:
I think that would be an improvement yes!